packages/dialog/src/vaadin-dialog.d.ts

```diff
@@ -102,6 +109,7 @@ export type DialogEventMap = DialogCustomEventMap & HTMLElementEventMap;
  *
  * @fires {CustomEvent} resize - Fired when the dialog resize is finished.
  * @fires {CustomEvent} opened-changed - Fired when the `opened` property changes.
+ * @fires {CustomEvent} closed - Fired when the overlay is closed.
```

Comment:
This line is still different compared to `.js` file: it says "overlay is closed" vs "dialog is closed".